protocol/message.go

```diff
@@ -186,6 +175,7 @@ func NewRegistrationProof(ap *m.AuthenticationPath, str *DirSTR,
 // NewKeyLookupProof creates the response message a CONIKS directory
 // sends to a client upon a KeyLookupRequest,
 // and returns a Response containing a DirectoryProof struct.
+// The length of `AP` and `STR` must to be equal to 1.
```

Comment:
This is only the case for registrations and key lookups. Monitoring and `LookupInEpoch`s return more than one auth path and/or STR. And even registrations are going to return multiple auth paths and STRs when we implement prior history verification.